include/mbedtls/ctr_drbg.h

```diff
@@ -209,7 +215,8 @@ void mbedtls_ctr_drbg_init( mbedtls_ctr_drbg_context *ctx );
  *   entropy sources).
  *
  * \p f_entropy is always called with a buffer size equal to the entropy
- * length described in the documentation of mbedtls_ctr_drbg_set_entropy_len().
+ * length. The entropy length is initially #MBEDTLS_CTR_DRBG_ENTROPY_LEN
```

Comment:
Reading this as a user I might think that calling `mbedtls_ctr_drbg_set_entropy_len()` after `mbedtls_ctr_drbg_init()` but before `mbedtls_ctr_drbg_seed()`, I can change the amount of entropy gathered during seeding. But this is not the case.

By the way, this behaviour seems like a bug to me. It is debatable if we want to fix it on the backport branches, but I think we should fix this on the development branch. (It seems to be a single line fix.)

Reply:
Good point, I clarified.

And I definitely don't want to change the behavior in LTS branches. Users might be using CTR_DRBG to get repeatable pseudorandom output. But since I've changed the behavior in development, I agree that it would be good to fix this too.

Reply:
And HMAC_DRBG has the same quirk, which I hadn't noticed, so I'll to fix the documentation accordingly in the LTS branches. This is getting to be quite the scope creep, but I think it's worth it to get it all out.